docs/sources/reference/config-blocks/import.file.md

```diff
@@ -41,6 +44,21 @@ The following arguments are supported:
 
 This example imports a module from a file and instantiates a custom component from the import that adds two numbers:
 
+{{< collapse title="main.alloy" >}}
```

Comment:
I'm not sure that using collapse here is really the best way to present the examples. What's the reason for choosing collapse over something simpler like a heading?

![image](https://github.com/user-attachments/assets/c1c0981b-8a7a-4d43-83bf-59338cba00e7)

Reply:
+1 I like the expanded version. I don't think there are good rules on when to use collapsible.

Reply:
Generally, the collapsed sections should be reserved for information that is extra to the flow... for example a legacy Agent Flow configuration. For examples I would prefer that we expand them out and use a more standard structured approach. There are multiple examples here, so the Example heading should be a little more descriptive. It's more work than I can put into the suggestions... so I can push a change to the PR or do you want to tweak things @wildum ?

Reply:
I made a few changes, @clayton-cornell feel free to tweak it via suggestions or to push changes directly if that's better